src/components/utils/unit_tests/string.test.js

```diff
@@ -1,4 +1,3 @@
-/* global expect */
```

Comment:
This would throw an error 'expect' is already defined as a built-in global variable. Expect is already defined as global in `.eslintrc`

Reply:
That `expect` is for `jest`. The value `false` indicates that it's not rewritable. As we add more jest unit tests we will probably add `eslint-plugin-jest` but for now I think it's OK to have it in the globals.